packages/services/f-feature-management/src/vue/services/features.service.js

```diff
@@ -41,28 +42,19 @@ function createSettings (store, serviceStore, cookies, analytics, logger) {
         onTrack: (key, variant) => trackExperiment(key, variant, analytics)
     };
 
-    if (logger) {
-        settings.logger = logger;
+    if (log) {
+        settings.log = log;
     }
 
     const initialConfigAsJson = serviceStore && serviceStore.configJson;
     if (initialConfigAsJson && initialConfigAsJson.features) {
-        logger.logInfo('[FeaturesService] Initialising with stored config');
+        log.info('[FeaturesService] Initialising with stored config', loggingTags);
         settings.initialConfigAsJson = initialConfigAsJson;
     }
 
     return settings;
 }
 
-function createLoggerWrapper (store, logger) {
```

Comment:
ℹ️ - Refactored this out, as the store is no longer required - I assume this is the reason for the wrapper.

Reply:
Yes, this is ancient. Good call.